libraries/session-storage/impl/src/main/sqldelight/migrations/1.sqm

```diff
@@ -1 +1,3 @@
+-- Migrate DB from version 1
+
```

Comment:
Same comment as before. This is redundant: Sqldelight explains 1.sqm == migration from version 1.
No problem to keep the comment. Just mentioning that we'll have one more thing to enforce in the future (we'll have to ensure that all *.sqm files have this header)